src/Debug/skeletonViewer.ts

```diff
@@ -560,13 +560,14 @@ export class SkeletonViewer {
         }
     }
 
-    private getAbsoluteRestPose(bone: Nullable<Bone>, matrix: Matrix) {
+    /** function to get the absolute bind pose of a bone by accumulating transformations up the bone hierarchy. */
+    private getAbsoluteBindPoseToRef(bone: Nullable<Bone>, matrix: Matrix) {

```

Comment:
nit: private functions should have a leading underscore in their name (I think we missed this one in the previous PR).

Reply:
Yes, as it is computing an absolute bind pose matrix it should start by using the local bind matrix, not the rest pose.